.github/workflows/docker-build.yml

```diff
@@ -0,0 +1,30 @@
+name: docker-build
+on: [push]
+
+env:
+  PROJECT_ID: docker-images-180022
+  IMAGE_NAME: gcr.io/docker-images-180022/apps/samson_secret_puller
```

Comment:
should this be hidden to public?

Reply:
probably, just can't see what ENV is available on this repo.. https://github.com/zendesk/samson_secret_puller#release-to-docker-hub mentions "For zendesk: image is pushed to GCR" so I'll take a leap of faith